notebooks/llm-rag-langchain/llm-rag-langchain-genai.ipynb

```diff
@@ -880,19 +880,49 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": null,
    "id": "d0bab20b",
    "metadata": {},
    "outputs": [],
    "source": [
+    "from transformers import AutoTokenizer\n",
+    "from openvino_tokenizers import convert_tokenizer\n",
+    "import openvino as ov\n",
+    "\n",
     "export_command_base = \"optimum-cli export openvino --model {} --task text-classification\".format(rerank_model_configuration[\"model_id\"])\n",
     "export_command = export_command_base + \" \" + str(rerank_model_id.value)\n",
     "\n",
     "display(Markdown(\"**Export command:**\"))\n",
     "display(Markdown(f\"`{export_command}`\"))\n",
     "\n",
     "if not Path(rerank_model_id.value).exists():\n",
-    "    optimum_cli(rerank_model_configuration[\"model_id\"], str(rerank_model_id.value), show_command=False, additional_args={\"task\": \"text-classificaton\"})"
+    "    optimum_cli(rerank_model_configuration[\"model_id\"], str(rerank_model_id.value), show_command=False, additional_args={\"task\": \"text-classificaton\"})\n",
+    "    hf_tokenizer = AutoTokenizer.from_pretrained(rerank_model_configuration[\"model_id\"], trust_remote_code=True)\n",
+    "    ov_tokenizer, ov_detokenizer = convert_tokenizer(hf_tokenizer, with_detokenizer=True, number_of_inputs=2)\n",
```

Comment:
@openvino-dev-samples , I've asked advice @pavel-esir  and @apaniukov  regarding how to better resolve second input addition in tokenizer. There is no need to complete tokenizer conversion: detokenizer remains the same, for converted tokenizer you can apply helper function from openvino tokenizers library https://github.com/openvinotoolkit/openvino_tokenizers/blob/master/python/openvino_tokenizers/tokenizer_transformations.py#L281

Reply:
My method is provided by @pavel-esir, could you help to check ?